### ISSUE TITLE
Visualization of the segmentation results

### ISSUE THREAD

**Issue by siinem** [DrSleep/tensorflow-deeplab-resnet#145]
Dear @DrSleep,
I am so new at the subject, and i wanted to ask, how do you obtain the visualization of input images, mask, prediction, such as presented in here: https://github.com/DrSleep/tensorflow-deeplab-resnet/tree/batch-norm . Would you share the code to observe the segmentation results? 

I have fine tuned the model on my own dataset and the performance in terms of mIoU was around 0.20 both for training and testing images. May you have any suggestion to improve the result?  Batch size is 4 and i used the default learning rate at the original implementation. I guess that i should investigate the best hyperparameters for my dataset, besides i need to see the segmentation results.
Thanks in advance
